samples/start-app/python-fastapi/app/usecase/openid/activate.py

```diff
@@ -0,0 +1,15 @@
+from app.dependencies import get_config
+from app.usecase.activate import activate_usecase
+import secrets, urllib.parse
+
+oauth_scopes = [
+    'read_channel_localization',
```

Comment:
we could imagine a way to merge oauth_scopes from activate and openid activate so we would not have to duplicate it. I am not certain to handle python well enough to offer an example tho